help(spec-support): re-organize the section (#151)
bump-sh/docs: src/_help/quick-start/index.md

You can setup [Custom Domains](/help/custom-domains), to make your doc available behind your own URL.
Several branding options are available: from your logo to [Meta Images](/help/meta-images) (that helps sharing your doc in a fancy way).

> We also support [Documentation Topics](/help/enhance-documentation-content/topics/), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.

Reviewer: ```suggestion
We also support [Documentation Topics](/help/enhance-documentation-content/topics/), which help you add rich (con)text to your API doc, sharing the information your community needs.
```
